From 7fafa3ccda857a8d65259e50060f5a6bcb5e0a0a Mon Sep 17 00:00:00 2001
From: Stef Walter <stefw@gnome.org>
Date: Mon, 30 Apr 2012 20:23:37 +0000
Subject: Align keyids with name/signatures in key properties

 * In the table for names/signatures, align the keyid with the
   name/userid markup
---
diff --git a/pgp/seahorse-pgp-key-properties.c b/pgp/seahorse-pgp-key-properties.c
index dafd3bf..019d945 100644
--- a/pgp/seahorse-pgp-key-properties.c
+++ b/pgp/seahorse-pgp-key-properties.c
@@ -532,13 +532,17 @@ do_names (SeahorseWidget *swidget)
                                                      "gicon", UIDSIG_ICON, NULL);
 
         /* TRANSLATORS: The name and email set on the PGP key */
+        renderer = gtk_cell_renderer_text_new ();
+        g_object_set (renderer, "yalign", 0.0, "xalign", 0.0, NULL);
         gtk_tree_view_insert_column_with_attributes (GTK_TREE_VIEW (widget), 
-                                                     -1, _("Name/Email"), gtk_cell_renderer_text_new (), 
+                                                     -1, _("Name/Email"), renderer,
                                                      "markup", UIDSIG_NAME, NULL);
 
         /* The signature ID column */
+        renderer = gtk_cell_renderer_text_new ();
+        g_object_set (renderer, "yalign", 0.0, "xalign", 0.0, NULL);
         gtk_tree_view_insert_column_with_attributes (GTK_TREE_VIEW (widget), 
-                                                     -1, _("Signature ID"), gtk_cell_renderer_text_new (), 
+                                                     -1, _("Signature ID"), renderer,
                                                      "text", UIDSIG_KEYID, NULL);
     }
 
--
cgit v0.9.0.2
